tests/v2/validation/rbac/rbac_test.go

```diff
@@ -59,6 +60,19 @@ func (rb *RBTestSuite) SetupSuite() {
 
 }
 
+func (rb *RBTestSuite) CheckRBACResourceExistsForUser(client *rancher.Client, collection *v1.SteveCollection, userName string) (bool, error) {
```

Comment:
this is a public function, will it be used elsewhere? either way, can you move it to rbac.go?

Reply:
agreed

Reply:
This was previously in rbac.go but I wanted it to be in the rbac_test.go as when the rbac_test.go moves to integration, we do not want to replicate rbac.go to integration test and we want the helper functions that are needed to be extensions, so its available for both integration and validation tests. But since this is a special case, this was added in rbac_test.go.
but @Priyashetty17 can you make this a private function instead? 

cc @igomez06